.gitignore

```diff
@@ -1,76 +1,26 @@
-# Logs
-*.log
+node_modules/
+.npm
+
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 lerna-debug.log*
 
-# Runtime data
-pids
-*.pid
-*.seed
-*.pid.lock
-
-# Directory for instrumented libs generated by jscoverage/JSCover
-lib-cov
-
-# Coverage directory used by tools like istanbul
-.coverage
-.jest-cache
-# nyc test coverage
-.nyc_output
-
-# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
-.grunt
-
-# Bower dependency directory (https://bower.io/)
-bower_components
-
-# node-waf configuration
-.lock-wscript
-
-# Compiled binary addons (http://nodejs.org/api/addons.html)
-build/Release
-
-# Dependency directories
-node_modules/
-jspm_packages/
-
-# Typescript v1 declaration files
-typings/
-
-# Optional npm cache directory
-.npm
-
-# Optional eslint cache
 .eslintcache
 
-# Optional REPL history
-.node_repl_history
-
-# Output of 'npm pack'
-*.tgz
-
-# cache and distribution folders
 .parcel-cache
 .cache/
 .now
 dist/
 storybook-static
-# Mac files
-.DS_Store
```

Comment:
We should keep this line: 
![image](https://user-images.githubusercontent.com/51168865/131964424-b0dc2143-e8e6-4892-bc11-176d252ca17c.png)

Reply:
Those should be in your global gitignore for your system IMO but sure, can bring it back